ServiceUseExtensionTest suffers from intermittent failure

CI builds regularly fail due to some timing bug in ServiceUseExtensionTest or the ServiceUseExtension the test is testing.

We need to solve this before any release.

```
2020-04-03T07:00:17.5195056Z [INFO] --- bnd-testing-maven-plugin:5.0.1:testing (testing) @ org.osgi.test.junit5 ---
2020-04-03T07:00:17.5253098Z [INFO] Matching glob *
2020-04-03T07:00:17.5343099Z [INFO] Bnd inferred -runee: JavaSE-1.8
2020-04-03T07:00:22.5700497Z TEST org.osgi.test.junit5.service.ServiceUseExtensionTest#successWhenServiceWithTimeout() <<< ERROR: 
2020-04-03T07:00:22.5701228Z The following assertion failed:
2020-04-03T07:00:22.5704550Z 1) [Services [{org.osgi.test.junit5.types.Foo}={service.id=36, service.bundleid=14, service.scope=singleton, case=successWhenServiceWithTimeout}]] 
2020-04-03T07:00:22.5704896Z Expecting:
2020-04-03T07:00:22.5705161Z  <2>
2020-04-03T07:00:22.5705450Z to be equal to:
2020-04-03T07:00:22.5705706Z  <1>
2020-04-03T07:00:22.5705946Z but was not.
2020-04-03T07:00:22.5707339Z at ServiceUseExtensionTest.successWhenServiceWithTimeout(ServiceUseExtensionTest.java:211)
2020-04-03T07:00:22.5760010Z 
2020-04-03T07:00:22.5763277Z org.assertj.core.api.SoftAssertionError: 
2020-04-03T07:00:22.5763884Z The following assertion failed:
2020-04-03T07:00:22.5766862Z 1) [Services [{org.osgi.test.junit5.types.Foo}={service.id=36, service.bundleid=14, service.scope=singleton, case=successWhenServiceWithTimeout}]] 
2020-04-03T07:00:22.5767226Z Expecting:
2020-04-03T07:00:22.5767526Z  <2>
2020-04-03T07:00:22.5767790Z to be equal to:
2020-04-03T07:00:22.5768083Z  <1>
2020-04-03T07:00:22.5768345Z but was not.
2020-04-03T07:00:22.5768657Z at ServiceUseExtensionTest.successWhenServiceWithTimeout(ServiceUseExtensionTest.java:211)
2020-04-03T07:00:22.5773562Z 
2020-04-03T07:00:22.5803562Z 	at org.assertj.core.error.AssertionErrorCreator.multipleSoftAssertionsError(AssertionErrorCreator.java:75)
2020-04-03T07:00:22.5804029Z 	at org.assertj.core.api.SoftAssertions.assertAll(SoftAssertions.java:166)
2020-04-03T07:00:22.5810325Z 	at org.osgi.test.junit5.service.ServiceUseExtensionTest.successWhenServiceWithTimeout(ServiceUseExtensionTest.java:225)
2020-04-03T07:00:22.5811224Z 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2020-04-03T07:00:22.5811733Z 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2020-04-03T07:00:22.5812114Z 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2020-04-03T07:00:22.5812599Z 	at java.lang.reflect.Method.invoke(Method.java:498)
2020-04-03T07:00:22.5824576Z 	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:515)
2020-04-03T07:00:22.5825198Z 	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:115)
2020-04-03T07:00:22.5825709Z 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$6(TestMethodTestDescriptor.java:171)
2020-04-03T07:00:22.5826032Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:72)
2020-04-03T07:00:22.5826646Z 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:167)
2020-04-03T07:00:22.5827017Z 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:114)
2020-04-03T07:00:22.5828240Z 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:59)
2020-04-03T07:00:22.5832055Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:105)
2020-04-03T07:00:22.5868638Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:72)
2020-04-03T07:00:22.5870120Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:95)
2020-04-03T07:00:22.5872199Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:71)
2020-04-03T07:00:22.5873141Z 	at java.util.ArrayList.forEach(ArrayList.java:1257)
2020-04-03T07:00:22.5876731Z 	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
2020-04-03T07:00:22.5878984Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:110)
2020-04-03T07:00:22.5881491Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:72)
2020-04-03T07:00:22.5885064Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:95)
2020-04-03T07:00:22.5886878Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:71)
2020-04-03T07:00:22.5889406Z 	at java.util.ArrayList.forEach(ArrayList.java:1257)
2020-04-03T07:00:22.5893088Z 	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
2020-04-03T07:00:22.5895251Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:110)
2020-04-03T07:00:22.5897357Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:72)
2020-04-03T07:00:22.5899267Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:95)
2020-04-03T07:00:22.5901145Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:71)
2020-04-03T07:00:22.5905044Z 	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:32)
2020-04-03T07:00:22.5907136Z 	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
2020-04-03T07:00:22.5909367Z 	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:51)
2020-04-03T07:00:22.5911781Z 	at aQute.tester.bundle.engine.BundleDescriptor.executeChild(BundleDescriptor.java:49)
2020-04-03T07:00:22.5914208Z 	at aQute.tester.bundle.engine.BundleEngine.lambda$executeBundle$7(BundleEngine.java:120)
2020-04-03T07:00:22.5916212Z 	at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
2020-04-03T07:00:22.5918063Z 	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:175)
2020-04-03T07:00:22.5919119Z 	at java.util.Iterator.forEachRemaining(Iterator.java:116)
2020-04-03T07:00:22.5921145Z 	at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
2020-04-03T07:00:22.5922953Z 	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:482)
2020-04-03T07:00:22.5924919Z 	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472)
2020-04-03T07:00:22.5926422Z 	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
2020-04-03T07:00:22.5928071Z 	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
2020-04-03T07:00:22.5929383Z 	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
2020-04-03T07:00:22.5931674Z 	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:485)
2020-04-03T07:00:22.5933340Z 	at aQute.tester.bundle.engine.BundleEngine.executeBundle(BundleEngine.java:120)
2020-04-03T07:00:22.5935934Z 	at aQute.tester.bundle.engine.BundleEngine.lambda$executeBundle$8(BundleEngine.java:133)
2020-04-03T07:00:22.5937185Z 	at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
2020-04-03T07:00:22.5938664Z 	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
2020-04-03T07:00:22.5940049Z 	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:175)
2020-04-03T07:00:22.5941739Z 	at java.util.Iterator.forEachRemaining(Iterator.java:116)
2020-04-03T07:00:22.5943505Z 	at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
2020-04-03T07:00:22.5945079Z 	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:482)
2020-04-03T07:00:22.5946541Z 	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472)
2020-04-03T07:00:22.6039062Z 	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
2020-04-03T07:00:22.6041598Z 	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
2020-04-03T07:00:22.6043013Z 	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
2020-04-03T07:00:22.6044491Z 	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:485)
2020-04-03T07:00:22.6046289Z 	at aQute.tester.bundle.engine.BundleEngine.executeBundle(BundleEngine.java:133)
2020-04-03T07:00:22.6047895Z 	at aQute.tester.bundle.engine.BundleEngine.lambda$execute$5(BundleEngine.java:100)
2020-04-03T07:00:22.6049353Z 	at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
2020-04-03T07:00:22.6050892Z 	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
2020-04-03T07:00:22.6052377Z 	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:175)
2020-04-03T07:00:22.6053511Z 	at java.util.Iterator.forEachRemaining(Iterator.java:116)
2020-04-03T07:00:22.6055386Z 	at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
2020-04-03T07:00:22.6056797Z 	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:482)
2020-04-03T07:00:22.6058233Z 	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472)
2020-04-03T07:00:22.6059786Z 	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
2020-04-03T07:00:22.6061395Z 	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
2020-04-03T07:00:22.6062758Z 	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
2020-04-03T07:00:22.6064214Z 	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:485)
2020-04-03T07:00:22.6065472Z 	at aQute.tester.bundle.engine.BundleEngine.execute(BundleEngine.java:100)
2020-04-03T07:00:22.6067141Z 	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:220)
2020-04-03T07:00:22.6069140Z 	at org.junit.platform.launcher.core.DefaultLauncher.lambda$execute$6(DefaultLauncher.java:188)
2020-04-03T07:00:22.6071025Z 	at org.junit.platform.launcher.core.DefaultLauncher.withInterceptedStreams(DefaultLauncher.java:202)
2020-04-03T07:00:22.6072723Z 	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:181)
2020-04-03T07:00:22.6074566Z 	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:128)
2020-04-03T07:00:22.6075684Z 	at aQute.tester.junit.platform.Activator.test(Activator.java:352)
2020-04-03T07:00:22.6076902Z 	at aQute.tester.junit.platform.Activator.automatic(Activator.java:317)
2020-04-03T07:00:22.6078044Z 	at aQute.tester.junit.platform.Activator.run(Activator.java:217)
2020-04-03T07:00:22.6079457Z 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
2020-04-03T07:00:22.6080589Z 	at aQute.launcher.Launcher.launch(Launcher.java:412)
2020-04-03T07:00:22.6081548Z 	at aQute.launcher.Launcher.run(Launcher.java:170)
2020-04-03T07:00:22.6082712Z 	at aQute.launcher.Launcher.main(Launcher.java:148)
2020-04-03T07:00:22.6085020Z 	at aQute.launcher.pre.EmbeddedLauncher.executeWithRunPath(EmbeddedLauncher.java:154)
2020-04-03T07:00:22.6190393Z 	at aQute.launcher.pre.EmbeddedLauncher.findAndExecute(EmbeddedLauncher.java:119)
2020-04-03T07:00:22.6191016Z 	at aQute.launcher.pre.EmbeddedLauncher.main(EmbeddedLauncher.java:48)
2020-04-03T07:00:22.8481726Z 
2020-04-03T07:00:22.8482490Z Test run finished after 2742 ms
2020-04-03T07:00:22.8483390Z [         9 containers found      ]
2020-04-03T07:00:22.8484056Z [         0 containers skipped    ]
2020-04-03T07:00:22.8484506Z [         9 containers started    ]
2020-04-03T07:00:22.8484795Z [         0 containers aborted    ]
2020-04-03T07:00:22.8485248Z [         9 containers successful ]
2020-04-03T07:00:22.8485740Z [         0 containers failed     ]
2020-04-03T07:00:22.8486189Z [        26 tests found           ]
2020-04-03T07:00:22.8486640Z [         0 tests skipped         ]
2020-04-03T07:00:22.8487019Z [        26 tests started         ]
2020-04-03T07:00:22.8487493Z [         0 tests aborted         ]
2020-04-03T07:00:22.8487913Z [        25 tests successful      ]
2020-04-03T07:00:22.8488376Z [         1 tests failed          ]
2020-04-03T07:00:22.8488676Z 
2020-04-03T07:00:22.9000070Z [INFO] 1 Error(s)
2020-04-03T07:00:22.9018694Z [ERROR] Error   : Exit code remote process 1: /opt/hostedtoolcache/jdk/8.0.242/x64/bin/java -cp /home/runner/work/osgi-test/osgi-test/org.osgi.test.junit5/target/test/tmp/testing/test/cnf/cache/5.0.1/bnd-cache/biz.aQute.launcher/biz.aQute.launcher.pre.jar -Dlauncher.properties=/home/runner/work/osgi-test/osgi-test/org.osgi.test.junit5/target/test/tmp/testing/test/generated/launch2633557528947728349.properties -ea -Dlauncher.runpath=/home/runner/.m2/repository/ch/qos/logback/logback-classic/1.2.0/logback-classic-1.2.0.jar,/home/runner/.m2/repository/ch/qos/logback/logback-core/1.2.0/logback-core-1.2.0.jar,/home/runner/.m2/repository/org/apache/felix/org.apache.felix.logback/1.0.2/org.apache.felix.logback-1.0.2.jar,/home/runner/.m2/repository/org/slf4j/slf4j-api/1.7.22/slf4j-api-1.7.22.jar,/home/runner/.m2/repository/org/eclipse/platform/org.eclipse.osgi/3.15.100/org.eclipse.osgi-3.15.100.jar,/home/runner/work/osgi-test/osgi-test/org.osgi.test.junit5/target/test/tmp/testing/test/cnf/cache/5.0.1/bnd-cache/biz.aQute.launcher/biz.aQute.launcher-5.0.1.jar aQute.launcher.pre.EmbeddedLauncher
```

--- a/org.osgi.test.junit5/src/test/java/org/osgi/test/junit5/service/ServiceUseExtensionTest.java
+++ b/org.osgi.test.junit5/src/test/java/org/osgi/test/junit5/service/ServiceUseExtensionTest.java
@@ -208,7 +208,7 @@ public class ServiceUseExtensionTest {
 				.getTrackingCount())
 				.as("Services %s", it.getExtension()
 					.getServiceReferences())
-				.isEqualTo(1);
+				.isGreaterThan(0);
 			softly.assertThat(it.getExtension()
 				.getCardinality())
 				.isEqualTo(1);

